src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -47,6 +61,81 @@ public void run() {
 				}
 			}
 		}
+		if ( inputValidation( all, testReport, list ) ) {
+			try {
+				final ReplayResult replayResult = ReplayResultProvider.load( testReport );
+
+				if ( !replayResult.containsChanges() ) {
+					logger.warn( "The test report has no differences." );
+					return;
+				}
+
+				try {
+					final LoadShouldIgnoreWorker loadShouldIgnoreWorker =
+							new LoadShouldIgnoreWorker( NopCounter.getInstance(), applier -> ignoreApplier = applier );
+					ignoreApplier = loadShouldIgnoreWorker.load();
+					logger.info( "The recheck ignore file loaded successfully." );
```

Comment:
The recheck ignore file **was** loaded successfully.

Successfully loaded recheck ignore.